app/controllers/hotels_controller.rb

```diff
@@ -1,14 +1,68 @@
 class HotelsController < ApplicationController
-  before_action :set_hotel, only: [:show]
+  before_action :set_hotel, only: %i[show edit update destroy]
+  before_action :set_labels, only: %i[new edit]
+
   def index
-    @hotels = Hotel.all
-    @hotels_count = Hotel.count
+    @hotels = policy_scope(Hotel).all
+    @hotels_count = policy_scope(Hotel).count
   end
 
   def show; end
 
+  def new
+    authorize Hotel
+    @hotel = Hotel.new
+  end
+
+  def create
+    authorize Hotel
+    @hotel = current_user.hotels.create(hotel_params)
```

Comment:
if you create here there is no matter to save. You have already created it here. You have 2 ways
- Use construction build & save 
- use create in if and read a diff between `create` and `create!`